src/ui/components/CredCardTemplate/CredCardTemplate.scss

```diff
@@ -46,10 +46,10 @@
     opacity: 0.04;
   }
 
-  &.card-body-w3c-generic,
+  &.card-body-generic,
   &.card-body-acdc {
```

Comment:
If `.card-body-generic` is not in use anymore (as we're only using acdc from now on) I believe there is no reason to keep it and then assign one value for the background here to acdc and then again at line 103.